rust/ql/test/extractor-tests/types/types.expected

```diff
@@ -1,56 +1,83 @@
+| main.rs:25:5:25:18 | main | fn main() |
+| main.rs:27:32:32:1 | BlockExpr | () |
 | main.rs:28:17:28:17 | 4 | usize |
 | main.rs:29:21:29:21 | 2 | usize |
 | main.rs:30:17:30:18 | 10 | i32 |
+| main.rs:31:5:31:48 | $crate::io::_print | fn _print(Arguments<'_>) |
 | main.rs:31:5:31:48 | MacroExpr | () |
-| main.rs:31:5:31:48 | PathExpr | fn _print(Arguments<'_>) |
+| main.rs:31:14:31:47 | "Value {value:#width$.precision$}\\n" | &str |
 | main.rs:31:14:31:47 | BlockExpr | () |
 | main.rs:31:14:31:47 | CallExpr | () |
 | main.rs:31:14:31:47 | FormatArgsExpr | Arguments<'_> |
 | main.rs:31:14:31:47 | MacroExpr | Arguments<'_> |
+| main.rs:31:22:31:26 | value | i32 |
+| main.rs:31:29:31:33 | width | usize |
+| main.rs:31:36:31:44 | precision | usize |
+| main.rs:34:23:42:1 | BlockExpr | () |
 | main.rs:35:5:41:5 | IfExpr | () |
 | main.rs:35:8:35:8 | x | i32 |
 | main.rs:35:8:35:12 | ... < ... | bool |
 | main.rs:35:12:35:12 | 5 | i32 |
+| main.rs:35:14:37:5 | BlockExpr | () |
+| main.rs:36:9:36:25 | $crate::io::_print | fn _print(Arguments<'_>) |
 | main.rs:36:9:36:25 | MacroExpr | () |
-| main.rs:36:9:36:25 | PathExpr | fn _print(Arguments<'_>) |
+| main.rs:36:18:36:24 | "small\\n" | &str |
 | main.rs:36:18:36:24 | BlockExpr | () |
 | main.rs:36:18:36:24 | CallExpr | () |
 | main.rs:36:18:36:24 | FormatArgsExpr | Arguments<'_> |
 | main.rs:36:18:36:24 | MacroExpr | Arguments<'_> |
+| main.rs:37:12:41:5 | IfExpr | () |
 | main.rs:37:15:37:15 | x | i32 |
 | main.rs:37:15:37:20 | ... > ... | bool |
 | main.rs:37:19:37:20 | 10 | i32 |
+| main.rs:37:22:39:5 | BlockExpr | () |
+| main.rs:38:9:38:25 | $crate::io::_print | fn _print(Arguments<'_>) |
 | main.rs:38:9:38:25 | MacroExpr | () |
-| main.rs:38:9:38:25 | PathExpr | fn _print(Arguments<'_>) |
+| main.rs:38:18:38:24 | "large\\n" | &str |
 | main.rs:38:18:38:24 | BlockExpr | () |
 | main.rs:38:18:38:24 | CallExpr | () |
 | main.rs:38:18:38:24 | FormatArgsExpr | Arguments<'_> |
 | main.rs:38:18:38:24 | MacroExpr | Arguments<'_> |
+| main.rs:39:12:41:5 | BlockExpr | () |
+| main.rs:40:9:40:26 | $crate::io::_print | fn _print(Arguments<'_>) |
 | main.rs:40:9:40:26 | MacroExpr | () |
-| main.rs:40:9:40:26 | PathExpr | fn _print(Arguments<'_>) |
+| main.rs:40:18:40:25 | "medium\\n" | &str |
 | main.rs:40:18:40:25 | BlockExpr | () |
 | main.rs:40:18:40:25 | CallExpr | () |
 | main.rs:40:18:40:25 | FormatArgsExpr | Arguments<'_> |
 | main.rs:40:18:40:25 | MacroExpr | Arguments<'_> |
+| main.rs:44:47:49:1 | BlockExpr | bool |
 | main.rs:45:5:48:5 | MatchExpr | bool |
 | main.rs:45:11:45:14 | pair | (bool, bool) |
+| main.rs:46:13:46:16 | true | bool |
 | main.rs:46:22:46:22 | x | bool |
 | main.rs:47:14:47:18 | false | bool |
+| main.rs:51:14:53:1 | BlockExpr | () |
+| main.rs:52:5:52:11 | $crate::boxed::Box::new | fn new<[i32; 1]>([i32; 1]) -> Box<[i32; 1], Global> |
+| main.rs:52:5:52:11 | (no string representation) | fn into_vec<i32, Global>(Box<[i32], Global>) -> Vec<i32, Global> |
```

Comment:
@paldepind FYI, I ran into the same error due to the incomplete `toString` definition for `PathSegment`.